src/genz-malik.jl

```diff
@@ -138,6 +138,8 @@ function (g::GenzMalik{n,T})(f, a::SVector{n}, b::SVector{n}, norm=vecnorm) wher
         if divdiff > maxdivdiff
```

Comment:
We could add a `divdiff = divdiff < eps(T) ? zero(divdiff) : divdiff` above this to ignore negligible differences and choose from the half-widths instead, but I'm not sure if its worth the trouble.